syntax = "proto3";

package lvi;

import "protos/bigquery_options.proto";

message Address {
  string street = 1;
  string number = 2;
  string country = 3;
}

message Actor {
  int64 userId = 1 [(required) = true, (description) = "Identifier in the master table"];
  string email = 2 [(required) = false, (description) = "Email address of the actor"];
  Address address = 3;
}

